R/scale-.r

```diff
@@ -631,10 +631,14 @@ continuous_scale <- function(aesthetics, scale_name, palette, name = waiver(),
 #' @param limits A character vector specifying the data range for the scale.
 #   The limits control what levels are displayed in the plot, their order,
 #'  and the default order of their display in guides.
-#' @param labels \code{NULL} for no labels, \code{waiver()} for default
-#'   labels (labels the same as breaks), a character vector the same length
-#'   as breaks, or a named character vector whose names are used to match
-#'   replacement the labels for matching breaks.
+#' @param labels One of: \itemize{
```

Comment:
Not really. My general convention has been when there's a class of similar function, inherit documentation from the first in the alphabet, but it's not a big deal.